lib/membrane_h264_plugin/parser.ex

```diff
@@ -559,7 +558,7 @@ defmodule Membrane.H264.Parser do
           Membrane.Time.t(),
           :au | :nalu,
           stream_structure()
-        ) :: Buffer.t()
+        ) :: Buffer.t() | [Buffer.t()]
   defp wrap_into_buffer(access_unit, pts, dts, :au, output_stream_structure) do
```

Comment:
Just a NIT suggestion to the code you haven't changed - I don't think there is a need to assign the result to `buffer` and then immediately return that buffer